pyuploadcare/api_resources.py

```diff
@@ -231,12 +231,28 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
+    def copy(self, effects=None, target=None, make_public=None, store=None, pattern=None):
         """Creates File copy
```

Comment:
Since number of parameters are grew - we can use kwargs, so no need for positional paramaters. 
Just write the good docs and everything will be fine.